tests/sentry/sentry_metrics/test_batch.py

```diff
@@ -326,6 +326,104 @@ def test_extract_strings_with_multiple_use_case_ids():
     }
 
 
+@patch("sentry.sentry_metrics.consumers.indexer.batch.UseCaseID", MockUseCaseID)
+def test_extract_strings_with_invalid_mri():
+    bad_counter_payload = {
```

Comment:
Nit: I generally prefer having some sort of docstring describing what the intent of the test is. Especially if the test function is very long as in this case.

Reply:
Fixed, thank you!